crates/superposition_types/src/database/models.rs

```diff
@@ -0,0 +1,4 @@
+pub mod cac;
+
```

Comment:
How so?

Reply:
so, the formatter auto sorts them in alphabetical order provided the mods are in a consecutive order
just like [here](https://github.com/juspay/superposition/pull/309/files#r1891389481)
the correct order should have been
```
pub mod models;
#[cfg(feature = "diesel_derives")]
pub mod schema;
pub mod types;
```

but since you added the newlines in between it could not sort them